projects/public-search/src/app/patron-profile/patron-profile-document/patron-profile-document.component.scss

```diff
@@ -0,0 +1,25 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+dl {
+  margin-bottom: 0;
+}
+
+dl > dt,
+dl > dd {
+  margin-bottom: 0;
+}
```

Comment:
Is it specific for this view ? In my opinion, if we change this, it should be done in bootstrap customization for all `dl, dt, dd` elements ; not only for this view. If I were only my decision, I will remove these CSS rule (and the file).
PS : You can merge both css rules into one because they use the same definition:
```css
dl, dt, dd {
  margin-bottom: 0;
}
```

Reply:
done